File: KysectAcademyTask.Tests/FilesForTests/FiltersTests/RootDirectory/M3237/Anna Thompson/6. Знакомство с паттернами/20191225193215/Motherboard.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LABA_6__Computer_.Components
{
    public class Motherboard : LABA_6__Computer_.API.Models.Component
    {
        public Motherboard(string VendorModel) : base(VendorModel)
        {
        }

        public override string GetConfiguration() => $"Motherboard: Vendor+Model - {VendorModel};";
    }
}

File: KysectAcademyTask.Tests/FilesForTests/FiltersTests/RootDirectory/M3236/Betty Padilla/6. Знакомство с паттернами/20191216201118/Motherboard.java
package entity;

public class Motherboard
{
	String model;
	String vendor;

	public Motherboard(String model, String vendor) {
		this.model = model;
		this.vendor = vendor;
	}

	@Override
	public String toString() {
		return "Motherboard{" +
				"model='" + model + '\'' +
				", vendor='" + vendor + '\'' +
				'}';
	}
}
